_maps/map_files/templates/shelter_1.dmm

```diff
@@ -0,0 +1,110 @@
+//MAP CONVERTED BY dmm2tgm.py THIS HEADER COMMENT PREVENTS RECONVERSION, DO NOT REMOVE 
```

Comment:
Should probably re-save this map to get rid of the conversion fluff; our map merge tool might not play nice with it.